fhir-bulkimportexport-webapp/src/main/java/com/ibm/fhir/bulkcommon/BulkDataUtils.java

```diff
@@ -137,7 +137,7 @@ public static void listBuckets(AmazonS3 cosClient) {
     private static int getFhirResourceFromBufferReader(BufferedReader resReader, int numOfLinesToSkip, List<Resource> fhirResources) throws Exception {
         int exported = 0;
         int lineRed = 0;
-        while (resReader.ready()) {
+        while (true) {
```

Comment:
This should be changed so the logic is expressed as part of the loops condition. 
Convert to a do-while if necessary... 

Also - FYI - https://wiki.sei.cmu.edu/confluence/display/java/MSC01-J.+Do+not+use+an+empty+infinite+loop
it's something to avoid

Reply:
yeah, good point. I forgot I used the same pattern in the COS break tool

Reply:
done